### PR TITLE
category

### PR DESCRIPTION
**Pull request by JacobCrozier**
Uncomplete

### CODE REVIEW

File: server/routes/questionRoute.js [hunk 1]
```diff
@@ -2,44 +2,55 @@ const express = require('express');
...
         .then(ans => res.json(ans))
         .catch(err => res.status(400).json(err));
 });
 
-router.get('/:id', (req, res) => {
-    Question.find({_id: req.params.id})
-        .then(ans => res.send(ans))
-        .then(ans => res.json(ans))
+router.get('/:category', (req, res) => {
+    Question.find({ category: req.params.category })
```
Comment (aidanbridge) [inline server/routes/questionRoute.js]: Looks like the /id route has been removed. Is that right?

File: server/models/question.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const questionSchema = new Schema({
     role: Boolean,
     datetime: Date,
     upvotes: Number,
-    category: [String]
+    category: String
 });
 
-module.exports = mongoose.model('Question', questionSchema);
\ No newline at end of file
+module.exports = mongoose.model('Question', questionSchema);
```
Comment (aidanbridge) [inline server/models/question.js]: No new line EOF

File: src/components/answer/CreateAnswer.js [hunk 3]
```diff
@@ -21,13 +21,13 @@ export default class CreateAnswer extends React.Component {
...
-            answer_title: "",
-            author: "",
+            answer_title: '',
+            author: '',
             datetime: null
         });
     };
 
-    handleInputChange = (event) => {
+    handleInputChange = event => {
```
Comment (aidanbridge) [inline src/components/answer/CreateAnswer.js]: Can't remember what the coding convention says about parantheis

File: server/routes/questionRoute.js (hunk 1 above)
Comment (JacobCrozier) [reply server/routes/questionRoute.js]: will change my bad

File: src/components/answer/CreateAnswer.js (hunk 3 above)
Comment (j-blackmore) [reply src/components/answer/CreateAnswer.js]: we are dropping brackets if 1 parameter for ES6 functions

Comment (JacobCrozier) [reply src/components/answer/CreateAnswer.js]: for arrow fns with only one parameter, no brackets

**Pull request merged**